app/scripts/background.js

```diff
@@ -175,6 +175,23 @@ if (globalThis.stateHooks.onInstalledListener) {
   });
 }
 
+/**
+ * Trigger actions that should happen only when an update is available
+ */
+function onUpdateAvailable() {
```

Comment:
We have a remote feature flag to enable/disable the update mechanism.

Because we haven't been able to actually test the full upgrade mechanism we need to put this functionality behind that flag.